coffeeconnection/coffeeconnection.py

```diff
@@ -230,7 +231,7 @@ def main():
         coffeeconnection(slack, config, niceties)
         return 0
     except Exception as error:
-        LOGGER.error("Error: %s", str(error))
+        LOGGER.error("Error: %s\n%s", str(error), traceback.format_exc())
```

Comment:
Can be changed to `LOGGER.exception("Error: %s", str(error))`

Oh hi Ced :)

Reply:
hey :wave: :bacon: